tests/apps/verify-ppb/pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[tool.briefcase]
+project_name = "Hello PPB"
+bundle = "org.beeware"
+version = "0.0.1"
+url = "https://beeware.org/"
+license = "BSD license"
+author = 'Brutus'
+author_email = "contact@beeware.org"
+
+[tool.briefcase.app.verify-ppb]
+template = "../../.."
+formal_name = "Hello PPB"
+description = "A PPB test app"
+icon = "src/verify_ppb/resources/verify-ppb"
+sources = ['src/verify_ppb']
+requires = [
+    'ppb~=1.1',
+]
+
+
+[tool.briefcase.app.verify-ppb.macOS]
+requires = [
+    'std-nslog~=1.0.0'
+]
+
+[tool.briefcase.app.verify-ppb.linux]
```

Comment:
The non-macOS targets should be removed to keep this config completely on-theme.

Reply:
Done.